### PR TITLE
Add filters to log pages

### PR DESCRIPTION
**Pull request by gergo85**

### CODE REVIEW

File: modules/backend/models/accesslog/columns.yaml [hunk 1]
```diff
@@ -3,23 +3,34 @@
...
         type: timetense
         width: 160px
 
+    type:
+        label: backend::lang.access_log.type
+        invisible: true
+
+    ip_address:
+        label: backend::lang.access_log.ip_address
+        searchable: yes
```
Comment (LukeTowers) [inline modules/backend/models/accesslog/columns.yaml]: Can you make this one invisible as well?

Comment (gergo85) [reply modules/backend/models/accesslog/columns.yaml]: Of course :)

Comment (gergo85) [reply modules/backend/models/accesslog/columns.yaml]: I made it.

**Comment by gergo85** [comment]
@LukeTowers Is it correct?

File: modules/backend/models/accesslog/columns.yaml (hunk 1 above)
Comment (LukeTowers) [reply modules/backend/models/accesslog/columns.yaml]: On second review I didn't notice that it was already present and visible; please leave this field visible by default.

Comment (gergo85) [reply modules/backend/models/accesslog/columns.yaml]: Now It is visible by default.

**Comment by gergo85** [comment]
@LukeTowers Could I change something?

**Pull request merged**